content/guides/identity-management/quick-start/1-scaffold-application-code.md

```diff
@@ -0,0 +1,218 @@
+---
+type: quick-start
+hide_in_page_nav: true
+---
+
+# Scaffold application code
+
+Our first step in this guide is to create a local application to house our code
+and configuration data as we start to create the Okta and Box applications that
+will be needed.
+
+Depending on your language / framework preference, create a blank application,
+install the required dependencies, as well as all configuration and program
+files.
+
+Choose your preferred language / framework below to get started.
+
+<Grid columns='4'>
```

Comment:
Honestly I don't think this looks nice. I'd change it to 2 instead.

<img width="845" alt="Screenshot 2020-04-28 at 12 18 46" src="https://user-images.githubusercontent.com/7718/80481488-6eef8e80-894a-11ea-863c-c3b331f8a06b.png">

Reply:
Done